README.md

```diff
@@ -47,7 +47,7 @@ Check latest version:
 ```yaml
 steps:
 - uses: actions/checkout@v2
-- uses: actions/setup-node@v2
+- uses: actions/setup-node@v1
```

Comment:
```suggestion
- uses: actions/setup-node@v2.0.0
```

Reply:
Ah, `v2` gave me an error and I saw that it was still beta.

Reply:
You should merge the suggestion so they can accept it (hopefully) :)